apps/ledger-live-mobile/src/families/evm/EvmFeesStrategy.tsx

```diff
@@ -7,7 +7,7 @@ import { log } from "@ledgerhq/logs";
 import { InfiniteLoader } from "@ledgerhq/native-ui";
 import { AccountBridge } from "@ledgerhq/types-live";
 import BigNumber from "bignumber.js";
-import React, { useCallback, useEffect, useState } from "react";
+import { default as React, useCallback, useEffect, useState } from "react";
```

Comment:
Does it change anything ? The former way of importing seems more what you would expect

Reply:
It does not change anything, seems like an auto formatting related change. Will revert it to stay consistent with the rest of the codebase 👍

Reply:
6359fba04b8137e73944511e540c2e1bb40ba39a